packages/salesforcedx-vscode-soql/package.json

```diff
@@ -38,18 +38,19 @@
     "@salesforce/soql-data-view": "0.1.0",
     "@salesforce/soql-language-server": "0.7.1",
     "@salesforce/soql-model": "0.2.3",
-    "jsforce": "^1.10.0"
+    "@types/jsforce": "1.9.41",
+    "jsforce": "1.10.0"
```

Comment:
since it was '^1.10.0' previously I think we've been running '1.11.0'. Should we pin to that version instead of back to 1.10.0?

Reply:
Good point 👍 Done